packages/svelte/tests/validator/samples/inline-new-class-2/input.svelte

```diff
@@ -1,6 +1,8 @@
 <script>
-	class Foo {
-		foo = $state(0)
+	function bar() {
+		class Foo {
+			foo = $state(0)
+		}
+		const a = new Foo();
```

Comment:
Since these are just warnings, I suggest to enhance this test case:
- have a `class Foo` at the instance and module script level to see that it doesn't warn on it
- have a `new class` at the instance and module script level and inside a function to see that it doesn't warn in the module script but does warn in the other two cases

Reply:
Done